launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the LoadComposableNodes action."""
+
+from typing import List
+from typing import Optional
+
+import composition_interfaces.srv
+
+from launch.action import Action
+from launch.actions import RegisterEventHandler
+from launch.event_handlers.on_process_start import OnProcessStart
+from launch.events.process import ProcessStarted
+from launch.launch_context import LaunchContext
+import launch.logging
+from launch.utilities import perform_substitutions
+
+from .composable_node_container import ComposableNodeContainer
+
+from ..descriptions import ComposableNode
+from ..utilities import evaluate_parameters
+from ..utilities import to_parameters_list
+
+
+class LoadComposableNodes(Action):
+    """Action that loads composable ROS nodes into a running container."""
+
+    def __init__(
+        self,
+        *,
+        composable_node_descriptions: List[ComposableNode],
+        target_container: ComposableNodeContainer,
+        **kwargs,
+    ) -> None:
+        """
+        Construct a LoadComposableNodes action.
+
+        The container node is expected to provide a `~/_container/load_node` service for
+        loading purposes.
+        Loading will be performed sequentially.
+
+        :param composable_node_descriptions: descriptions of composable nodes to be loaded
+        :param target_container: the container to load the nodes into
+        """
+        self.__composable_node_descriptions = composable_node_descriptions
+        self.__target_container = target_container
+        self.__logger = launch.logging.get_logger(__name__)
```

Comment:
missing call to `super().__init__(**kwargs)`. Without this I see

```
launch_service.py          355 ERROR    Caught exception in launch (see debug for traceback): 'LoadComposableNodes' object has no attribute '_Action__condition'
```

Reply:
Oops. Fixed.